### PR TITLE
djds/pylint

### PR DESCRIPTION
**Pull request by djds**
- [recorder] Specify UTF-8 encoding
- [pty_] Avoid shadowning class name with copy() method argument
- [style] Add pylint exceptions

### CODE REVIEW

File: asciinema/asciicast/raw.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
 class writer(file_writer):
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
```
Comment (sickill) [inline asciinema/asciicast/raw.py:9]: Btw, how are you finding these? Running `pylint` manually, or you have pylint integration in your editor?

**Review by sickill — APPROVED** [review]

File: asciinema/pty_.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def copy(signal_fd: int) -> None:  # pylint: disable=too-many-branches
     os.waitpid(pid, 0)
 
 
-class signal_fd:
+class SignalFD:
```
[the remark this answers is not in the document]
Comment (djds) [reply asciinema/pty_.py:153]: I would usually stick to `PascalCase` class names unless there's an external reason for doing differently. Either way the issue here wasn't the name itself, but rather that `positional argument to `copy` was shadowing the `signal_fd` class:
```python
     def copy(signal_fd: int) -> None:  # pylint: disable=too-many-branches
         fds = [pty_fd, tty_stdin_fd, signal_fd]
         stdin_fd = pty.STDIN_FILENO
```

File: asciinema/asciicast/raw.py (hunk 1 above)
Comment (djds) [reply asciinema/asciicast/raw.py:9]: Both, but https://github.com/dense-analysis/ale is always running in the background.

Comment (sickill) [reply asciinema/asciicast/raw.py:9]: Got it, thanks. `ale` looks nice. Yeah, I love shellcheck, taught me better scripting practices.

File: asciinema/pty_.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sickill) [reply asciinema/pty_.py:153]: Right, I saw the name clash. Was just curious about your general suggestion about the context-managers. Thanks!

**Pull request merged**